rcl/test/CMakeLists.txt

```diff
@@ -254,8 +227,11 @@ function(test_target_function)
     LIBRARIES ${PROJECT_NAME} mimick
     AMENT_DEPENDENCIES ${rmw_implementation} "osrf_testing_tools_cpp" "test_msgs"
   )
+  # TODO(asorbini) Enable message timestamp tests for rmw_connextdds on Windows
+  # once clock incompatibilities are resolved.
   if(rmw_implementation STREQUAL "rmw_fastrtps_cpp" OR
-    rmw_implementation STREQUAL "rmw_fastrtps_dynamic_cpp")
+    rmw_implementation STREQUAL "rmw_fastrtps_dynamic_cpp" OR
+    (rmw_implementation STREQUAL "rmw_connextdds" AND NOT WIN32))
```

Comment:
My major question here is why this is different with `rmw_connextdds_cpp`, when it seemingly worked before with `rmw_connext_cpp`.  Do we have a good idea?

Reply:
the `rmw_connext_cpp` were being completely skipped, now they aren't skipped except than on Windows.
i.e.: it looks better now :smile: